scripts/build/createTag.sh

```diff
@@ -24,7 +24,7 @@ function usage()
   empty=$(echo $sfile | tr 'a-zA-z/0-9.:' ' ')
```

Comment:
Probably this script is no longer used, but I have tried to adapt it anyway.

NTC (informative)